### ISSUE TITLE
Firefox addon failed review

### ISSUE THREAD

**Issue by digitarald** [BabylonJS/Spector.js#120]
Firefox DevTools member here 👋🏻 – Just wanted to close the loop for an email you might have missed from the Firefox addons team.

Firefox DevTools has started to recommend Spector.js for WebGL debugging since we deprecated and removed the Shader inspector tool. Thanks so much for your great work!

Users just started to report that the addons page is down, so I followed up with the addons team. It looks like the last submission had original code attached that threw build errors, confirmed from 2 reviewers. The way to address this is uploading a new working version via AMO, making sure that you specify a node version and including a package-lock.json so build output matches.

Sorry for the extra work. I am happy to help if you need further information or help with anything.

**Comment by sebavan** [comment]
Ohhh, thanks a lot for the report, I will have a look on Monday and try to reupload everything. Sorry about that.

**Comment by follower** [comment]
FYI: The link https://addons.mozilla.org/en-US/firefox/addon/spector-js/#&gid=1&pid=2 from https://developer.mozilla.org/en-US/docs/Tools/Deprecated_tools still leads to a "Oops! We can’t find that page" page.

(FWIW @digitarald:

 * You might like to pass on to the team concerned that I don't think that 404ing a valid extension that has (temporarily) failed a review is particularly great UX.

 * Also, while off topic, as a data point, I've finally installed the Developer Edition of FF after using Waterfox for many years--specifically to debug WebAssembly/WebGL/Canvas code from the Godot game engine and discovering that all the related devtools have been removed without replacement hasn't made the best of impressions... I realise FF is stretched for resources but removing debugging tools for more advanced/complex technologies doesn't exactly help with the adoption of those technologies.

 * I'm also curious if there's a correlation between using these more advanced tools/technologies and disabling telemetry because I know *I* just spent a ridiculous amount of time trying to *really* disable FF telemetry before I discovered the devtools I was specifically wanting to use were removed.)
